### DISCUSSION TITLE
Integrate my dock-themeswitcher thingy

### DISCUSSION THREAD

**Discussion by teamcons** [pantheon-tweaks/pantheon-tweaks#249] [Ideas]
Hello

So with eOS 8 around the corner, a few people already started complaining about the new dock

To shift a fair bit the heat and expectations people could have, I did a bash script that allows to switch dock theme or revert to vanilla.
I am waiting on a PR that enable more option to be able to offer a Traditional theme and some more fully baked ones.

However if eOS do not support theming for the dock, i think it would make sense to offer my little utility along to pantheon-Tweaks ? That would avoid fragmentation of tweaking tools, and people who want that gets all in one place. Moreover the people who want theming wont pester Danielle and gang.

Im not great at Vala but i can maybe do a design, or cobble something together so theres a "Dock" page ?
(I am fine if yall have a better implementation and do not want it. It is basic anyway.)

**Comment by ryonakano** [comment]
I agree with providing a UI to change dock theme in Pantheon Tweaks.

I don't have a detailed plan yet, but regarding to the UI maybe creating a new dropdown in Appearance Pane would be enough instead of creating a new designated pane.
